exotica_python/src/pyexotica.cpp

```diff
@@ -960,8 +961,17 @@ PYBIND11_MODULE(_pyexotica, module)
     scene.def("get_model_joint_names", &Scene::GetModelJointNames);
     scene.def("get_model_state", &Scene::GetModelState);
     scene.def("get_model_state_map", &Scene::GetModelStateMap);
+    scene.def("get_tree_map", [](Scene& scene) {
+        std::map<std::string, std::shared_ptr<KinematicElement>> map_shared;
+        for (const auto& m : scene.GetTreeMap())
+        {
+            map_shared[m.first] = m.second.lock();
+        }
+        return map_shared;
+    });
```

Comment:
The `TreeMap` elements get out of scope after loading a scene. If they get exposed as a shared pointer, the user will end up holding a dangling pointer or even worse, blocking the scene editing process.

Expose only `get_tree_names`, extracting only the keys of the map.